#001 adding the landing page

ishimwebernard opened the pull request:
#### What does this PR do?
[what does this PR do?]
>This PR present the user with basic information about me, what I do and ways to connect with me through social media
>This PR redirects the user to other pages and the sign in page
#### Description of Task to be completed
[the tasks that you completed on the feature]
>added the method to reveal and hide the hamburger button for mobile version.
>added the animations that show up when the page loads
#### How should this be manually tested?
>Clone this repo with git clone https://github.com/ishimwebernard/bernard_portfolio.git
>Open the index.html file

#### What are the relevant Trello story?
>https://trello.com/c/mDacIl9u
#### Screenshots (if appropriate)
>

ishimwebernard committed:
adding the landing page HTML/CSS

ishimwebernard committed:
Editing and indexing

mukunzidd reviewed (changes requested):
- Change the entry file's name to index.html not kindex.html

mukunzidd reviewed (changes requested):
- Change the entry file's name to index.html not kindex.html
- Reduce font sizes all around to make elements and sections stand out. 
- Make sure that the hamburger menu is visible and functional on the mobile device screens

mukunzidd commented:
@ishimwebernard please treat a PR as your work presentation. 

```
[what does this PR do?]
I finished the landing page
```

This doesn't explain what you did. Please check the channel for a reference on how a PR description should be structured. 
Also remove the on the kstyles.css file 😢.

ishimwebernard committed:
Renamed the files and added some changes

ishimwebernard committed:
deleting unnecessary files

ishimwebernard committed:
Adding missing images

ishimwebernard committed:
Adding missing images 2

ishimwebernard committed:
Adding JS to reveal the Hamburger

ishimwebernard committed:
Adding animatins

mukunzidd commented:
What PR replaced this? You just closed it without telling me.

mukunzidd commented:
@ishimwebernard

ishimwebernard committed:
fix: LANDING PAGE

*. I added the responsiveness from 1024 to 425px
*. I updated the remote repo so, the footer images are rendering.
